src/pages/Statistics/StatisticsRegions.tsx

```diff
@@ -182,13 +182,15 @@ const StatisticsCities = () => {
       let regions = response.data as Region[];
       setRegions(
         regions
-        .sort((a: Region, b: Region) => a.regionName.localeCompare(b.regionName))
-        .map((item) => {
-          return {
-            label: item.regionName,
-            value: item.id,
-          };
-        })
+          .sort((a: Region, b: Region) =>
+            a.regionName.localeCompare(b.regionName)
```

Comment:
Why in StatisticsCities.ts similar call to localeCompare is on the same line as sort(), please make it consistent.